### PR TITLE
feat: use husky 6

### LINKED ISSUE #150: lint-staged task issue
Hey guys

Our teams depend on Husky 4 but we're now endng up with git-simple-hooks? It doesn't have the same features at all and things are breaking

To my knowledge Husky 4 isn't deprecated the author maintains it. We plan to move to Husky5 when the license will be reverted as it fits better.

If you want to introduce a new project, it would be better to name it `lint-staged-simple-git-hooks` and not break the existing.

Lastly the comparison that motivated the change is unfair IMHO. Git-simple-hooks size and perf were compared to husky 4 but if they're compared to Husky 5 it's roughly the same.

Cheers

### PR DESCRIPTION
**Pull request by typicode**
Husky 6 has been released and is MIT. It's only 6 kB now with the same main features as husky 4.
Fixes #150

### CODE REVIEW

File: packages/mrm-task-lint-staged/index.js [hunk 1]
```diff
@@ -179,22 +180,48 @@ module.exports = function task({ lintStagedRules }) {
...
 	install(packages);
+	// Install husky
+	husky.install();
+	// Set lint-staged config
+	husky.add('.husky/pre-commit', 'npx lint-staged');
 };
 
 module.exports.description = 'Adds lint-staged';
 module.exports.parameters = {
+	isYarn2: {
```
Comment (sapegin) [inline packages/mrm-task-lint-staged/index.js:220]: Could we use [this method](https://github.com/sapegin/mrm/blob/481a3161bc9c1a778a27b73cd746f4a4d756a41d/packages/mrm-core/src/npm.js#L275-L280) to autodetect Yarn 2? Or Yarn 2 and Yarn Berry aren't the same?

**Review by sapegin — COMMENTED** [review]
Thanks for the pull request! I only have one question ;-)

File: packages/mrm-task-lint-staged/index.js (hunk 1 above)
Comment (typicode) [reply packages/mrm-task-lint-staged/index.js:220]: I think we could use it. Yarn Berry and Yarn 2 are the same.

Comment (typicode) [reply packages/mrm-task-lint-staged/index.js:220]: Can I export it in https://github.com/sapegin/mrm/blob/481a3161bc9c1a778a27b73cd746f4a4d756a41d/packages/mrm-core/src/npm.js#L278-L285

Comment (typicode) [reply packages/mrm-task-lint-staged/index.js]: Updated, should be good now :)

**Review by sapegin — APPROVED** [review]
🦄

**Comment by typicode** [comment]
Thanks :)

**Pull request merged**